Assets/Character/Scripts/PlayerInput.cs

```diff
@@ -4,44 +4,44 @@ public static class PlayerInput
 {
     public static bool PressedRoll()
     {
-        return Input.GetKeyDown(KeyCode.C);
+        return Input.GetKeyDown(KeyCode.C) || Input.GetButtonDown("Roll");

```

Comment:
I can't tell if you're being sarcastic LOL but I'm gonna combine them into just GetButtonDown("Action") through the input manager.